src/citrine/informatics/design_candidate.py

```diff
@@ -132,8 +142,28 @@ def __init__(self, *, values: dict):
         pass
 
 
+class HierarchicalDesignMaterial(Serializable["HierarchicalDesignMaterial"]):
+    """Description of a designed material as a set of connected individual materials.
+
+    A hierarchical material contains a root material, containing identifiers and variables,
+    along any of sub-materials appearing in the history of the root.
+    Connections between the root and sub-materials are contained in the mixtures dictionary
+    that associates each material (by Citrine ID) with the ingredients that comprise it.
+    """
+
```

Comment:
It was a mistake to call this JSON field "terminal" in the backend, but its too late for that now. Hence why I am renaming it here.